### ISSUE TITLE
Form with disabled button

### ISSUE THREAD

**Issue by HugoGiraudel** [squizlabs/HTML_CodeSniffer#195]
Hello,

I hope this is not a duplicate. I had a look and could not find a similar issue.

We have a problem where a form containing a disabled button throws error `WCAG2AA.Principle3.Guideline3_2.3_2_2.H32.2`. 

> • Error: This form does not contain a submit button, which creates issues for those who cannot submit the form using the keyboard. Submit buttons are INPUT elements with type attribute "submit" or "image", or BUTTON elements with type "submit" or omitted/invalid.

I had a quick discussion about it with @heydon and it seems a bit agressive as a rule. Is there a way we could make it a bit more bulletproof? :)

**Comment by ironikart** [comment]
I've had a brief look at the code that was written to check this and it could probably be improved, however, setting a standard button or input with type="submit" to disabled shouldn't prevent it from being detected.

If possible, can you reply with the following:
- Browser & OS versions
- Failing code sample

The sniff that detects this uses a function that may not be available in older browsers (querySelectorAll), but has been available in the evergreens for some time now.

**Comment by HugoGiraudel** [comment]
I ran the tests through [pa11y](https://github.com/pa11y/pa11y), so I’m not entirely sure what set up is being used under the hood. I guess it’s a headless Chromium, so API support should not be an issue. 

I can see if I can give you a sample of code.
